### DISCUSSION TITLE
Sharing field Array with child component. Possible with useFormContext?

### DISCUSSION THREAD

**Discussion by MImranAsghar** [react-hook-form/react-hook-form#6384] [Q&A] [answered]
hey, I wanted to know that if I wanted to use same fieldArray in two different components (aka share all it's functions for the same instance of field array between different components, aka parent component and child component), how can I achieve that? I have used context for other fields but not sure how I can do the same with fieldArrays of React-hook-form.

The goal is to update the field array's fields in both parent and child component.

**Comment by MImranAsghar** [comment]
@bluebill1049 what do you think?

**Comment by bluebill1049** [comment, marked as answer]
not possible with FormContext at the moment, you will have to pass as props.
